### ISSUE TITLE
Unable to upgrade to the latest console version

### ISSUE THREAD

**Issue by kingsd041** [rancher/os#2243]
**RancherOS Version: (ros os version)**
v1.2.0-rc2
**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
AWS

**Steps to Reproduce:**
1. Launch rancherosv1.1.1 instance
2. Switch ubuntu console 
```
ros console switch ubuntu
```
3. Upgrade os and console to v1.2.0-rc2
Note `--upgrade-console` parameter
```
ros os upgrade --upgrade-console -i rancher/os:v1.2.0-rc2
```
4. Check the ubuntu console version
```
# system-docker ps | grep console
47ccad691909        rancher/os-ubuntuconsole:v1.1.0    "/usr/bin/ros entrypo"   2 minutes ago       Up 10 seconds                           console
```
**Results:**
Ubuntu console did not upgrade to the latest v1.2.0-rc2 version

**Comment by niusmallnan** [comment]
This should be a problem of the previous versions.
I will fix it in 1.2.0, and backport to 1.1.4.

@kingsd041  You can confirm like this:
```
ros os upgrade -i niusmallnan/os
# this can get the fixed ros command
```

Then you can upgrade to any version, and check the console.

**Comment by kingsd041** [comment]
The issue has been fixed.
I can use "--upgrade-console" to upgrade the console to the latest version.
